test/services/WhatYouOweServiceSpec.scala

```diff
@@ -333,38 +318,14 @@ class WhatYouOweServiceSpec extends TestSupport with FeatureSwitching with Charg
       }
     }
 
-    "with ReviewAndReconcilePoa" should {
-      def testGetWhatYouOweChargesList(ReviewReconcileEnabled: Boolean, financialDetails: FinancialDetailsModel,
-                                       expectedResult: WhatYouOweChargesList): Unit = {
-        if (ReviewReconcileEnabled) enable(ReviewAndReconcilePoa) else disable(ReviewAndReconcilePoa)
-        when(mockOutstandingChargesConnector.getOutstandingCharges(any(), any(), any())(any()))
-          .thenReturn(Future.successful(OutstandingChargesModel(List())))
-        when(mockFinancialDetailsService.getAllUnpaidFinancialDetails()(any(), any(), any()))
-          .thenReturn(Future.successful(List(financialDetails)))
-        TestWhatYouOweService.getWhatYouOweChargesList(isEnabled(ReviewAndReconcilePoa),
-          isEnabled(FilterCodedOutPoas),
-          isPenaltiesEnabled = isEnabled(PenaltiesAndAppeals),
-          mainChargeIsNotPaidFilter).futureValue shouldBe expectedResult
-      }
-      "return list including POA extra charges" in {
-//        testGetWhatYouOweChargesList(ReviewReconcileEnabled = true, financialDetails = financialDetailsReviewAndReconcile, expectedResult = whatYouOweWithReviewReconcileData)
-//        testGetWhatYouOweChargesList(ReviewReconcileEnabled = true, financialDetails = financialDetailsWithMixedData4, expectedResult = whatYouOweDataWithMixedData4Unfiltered)
-      }
-      "return list excluding POA extra charges" in {
-        testGetWhatYouOweChargesList(ReviewReconcileEnabled = false, financialDetails = financialDetailsReviewAndReconcile, expectedResult = whatYouOweEmpty)
-        testGetWhatYouOweChargesList(ReviewReconcileEnabled = false, financialDetails = financialDetailsWithMixedData4, expectedResult = whatYouOweDataWithMixedData4Filtered)
-      }
-    }
-
```

Comment:
I think the top one here is still valid. Just remove the enable/disable bit, and the test excluding POA extra charges